resources/authgear/templates/en/web/enter_password.html

```diff
@@ -21,17 +21,11 @@ <h1 class="primary-txt text-center margin-0 text-xl font-bold">
 
 <input type="hidden" name="x_stage" value="{{ $.AuthenticationStage }}">
 
-<input
-	tabindex="0"
-	autofocus
-	id="password"
-	class="input text-input primary-txt"
-	type="password"
-	autocomplete="current-password"
-	autocapitalize="none"
-	name="x_password"
-	placeholder="{{ template "password-placeholder" }}"
->
+{{ template "__password_input.html" (dict
+	"Name" "x_password"
+	"Type" "old-password"
+	"AutoFocus" true
+) }}
```

Comment:
We should also update the new parameter `Type` in `create_password.html` and `reset_password.html`.

Reply:
Fixed.